app/views/ladder/LadderView.coffee

```diff
@@ -159,6 +184,20 @@ module.exports = class LadderView extends RootView
     window.open url, if key.command then '_blank' else 'spectate'  # New tab for spectating specific matches
     #Backbone.Mediator.publish 'router:navigate', route: url
 
+  onClickSimulateAllButton: (e) ->
+    $.ajax
+      url: '/queue/scoring/loadTournamentSimulationTasks'
+      data:
+        originalLevelID: @level.get('original'),
+        levelMajorVersion: 0,
+        leagueID: @leagueID
+      type: 'POST'
+      parse: true
+      success: (res)->
+        console.log res
```

Comment:
emm... this is an async function, it might always return the same result. So if we want to do something here, maybe just show some alert or something, didn't think about it too much

Reply:
Is there any user experience or user feedback that needs to happen due to clicking this button?
What if the user clicks the button lots of times? Will it lead to the server overloading? Any risk?

Reply:
this button will only be visible for administrator. 
by the way, I'm going to move this function into a new tournament admin dashboard page, just leave it